libs/dh/metering-point/feature-child-overview/src/lib/dh-metering-point-child-overview.component.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'dh-metering-point-child-overview',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+    `,
+  ],
```

Comment:
suggestion: what do you think about keeping it short and sweet by default?

```suggestion
  styles: [':host { display: block; }'],
```

Reply:
I'd vote for having a separate scss file (for the DH app) unless the styling is a single line (like in this case). I don't get any IDE auto-complete and stuff when the styling is in the component class (though, this might be a matter of installing an extension, idn).

Reply:
The styles are adjusted to be single lines now, I think we should have a talk about this outside this pull-request.